itensor/mps/sweeps.h

```diff
@@ -486,24 +486,12 @@ operator<<(std::ostream& s, const Sweeps& swps)
     }
 
 void inline
-sweepnext(int &b, int &ha, int N, int min_b = 1)
+sweepnext(int &b, int &ha, int N, Args const& args = Args::global(), int min_b = 1)
```

Comment:
Should the `Args` come last? This may be breaking if someone input `min_b`.

We may want to make `min_b` input as an arg, and then have a wrapper function for backwards compatibility. For example:
```C++
void inline
sweepnext(int &b, int &ha, int N, Args const& args = Args::global())
  { // The new version
  }

void inline
sweepnext(int &b, int &ha, int N, int min_b)
  { // Creates an Arg, adds min_b to it, and calls the other sweepnext
  }
```

Reply:
But what if someone input args but not min_b?

Yes, I think have a wrapper function is a good idea.